### PR TITLE
Merge back master to develop

### PR DESCRIPTION
**Pull request by Scooletz**
Connects to #135

PR merges master to develop to properly align the packages versions

### CODE REVIEW

**Comment by Scooletz** [comment]
Ping @Particular/azure-maintainers for a review.

File: src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/app.config [hunk 1]
```diff
@@ -3,6 +3,10 @@
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
+        <assemblyIdentity name="Microsoft.WindowsAzure.Storage" publicKeyToken="31bf3856ad364e35" culture="neutral" />
```
Comment (SeanFeldman) [inline src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/app.config]: Not the same @SimonCropp, [range/versions are different](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/pull/140/files#diff-72ec67f87259888d5ed887536235f337R22) 😛 

> "0.0.0.0-6.2.2.0" newVersion="6.2.2.0"

**Comment by SeanFeldman** [comment]
[What] Simon said.

**Comment by Scooletz** [comment]
This is a backporting PR. Shouldn't change like this (which has no impact on running) tests should be raised and addressed in another PR @SeanFeldman ?

**Comment by SeanFeldman** [comment]
@Scooletz not following your question...

**Comment by Scooletz** [comment]
This PR is merging `master` to `develop`. `master` also contains this mistake. That's why I asked is there a reason for fixing it in the PR that has a single purpose: merge `master` back to `develop` @SeanFeldman 

Should I fix it here?

**Comment by SeanFeldman** [comment]
I really have no preferences.

**Comment by Scooletz** [comment]
I'm merging this as it's been created on github (no branch to pull) and follow up with a separate one.

**Comment by Scooletz** [comment]
I double checked the `app.config` and found no conflicting entries. Not issuing another PR then.

**Pull request merged**